src/Slick/Build.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Slick.Build
```

Comment:
What's the distinction between this module and the rest of slick? Is this just extra tools? If so we should clarify the name, as a beginner it would be a bit confusing when I need just `Slick` and when I need `Slick.Build`